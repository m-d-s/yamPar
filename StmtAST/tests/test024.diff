1,4c1
< Locals
<   int
<   VarIntro
<     "x"
---
> Invalid syntax at (1,1), int
