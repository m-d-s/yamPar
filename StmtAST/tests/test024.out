Call:   Top
  Call:   Stmt
    Call:   Decl
      Call:   Type
        Consumed token: <"int" at line 1 column 1>
      Return: Type
      Call:   Intros
        Call:   Var
          Consumed token: <<IDENT>: "x" at line 1 column 5>
        Return: Var
      Return: Intros
      Consumed token: <";" at line 1 column 6>
    Return: Decl
  Return: Stmt
  Consumed token: <<EOF> at line 1 column 40>
Return: Top
Locals
  int
  VarIntro
    "x"
